### PR TITLE
[Minor] improvement(docs): Optimize Trino related docs and fix some minor mistakes in docs

### PR DESCRIPTION
**Pull request by yuqi1129**
### What changes were proposed in this pull request?

Refine some descriptions and correct mistakes in Trino-related docs.

### Why are the changes needed?

Make users more aware of the documents.

### Does this PR introduce _any_ user-facing change?

N/A.

### How was this patch tested?

N/A.

### CODE REVIEW

File: docs/trino-connector/supported-catalog.md [hunk 1]
```diff
@@ -15,13 +15,12 @@ The catalogs currently supported by the Gravitino connector are as follows:
...
 
-Trino currently does not support creating Gravitino managed catalogs. 
-If you need to create a catalog, please refer to:
-- [Create a Catalog](../manage-metadata-using-gravitino.md#create-a-catalog)
+Trino currently does not support creating Gravitino managed catalogs, if you need to create a catalog, please refer to: [Create a Catalog](../manage-metadata-using-gravitino.md#create-a-catalog)
 
 ## Data type mapping between Trino and Gravitino
 
-All types of catalogs support the following data type conversions between Trino and Gravitino:
+Gravitino connector supports the following data type conversions between Trino and Gravitino currently. Depending on the detailed catalog, some data types conversion may not be supported for this specific catalog, for example
```
Comment (jerryshao) [inline docs/trino-connector/supported-catalog.md:22]: "for example" -> "for example, "

File: docs/trino-connector/supported-catalog.md [hunk 2]
```diff
@@ -36,8 +35,11 @@ All types of catalogs support the following data type conversions between Trino
...
 | DateType       | DATE       |
+| TimeType       | TIME       |
+| TimestampType  | TIMESTAMP  |
 | ArrayType      | ARRAY      |
 | MapType        | MAP        |
-| StructType     | ROW        |
\ No newline at end of file
+| StructType     | ROW        |
+
+For more about Trino data types, please refer to [Trino data types](https://trino.io/docs/current/language/types.html) and Gravity data types, please refer to [Gravity data types](../manage-metadata-using-gravitino.md#gravitino-table-column-type)
```
Comment (jerryshao) [inline docs/trino-connector/supported-catalog.md:45]: Typo: "Gravitino"

File: docs/trino-connector/configuration.md [hunk 3]
```diff
@@ -6,8 +6,8 @@ license: "Copyright 2023 Datastrato Pvt Ltd.
...
+| connector.name     | string | (none)                | The `connector.name` defines the name of Gravitino connector, this value is always 'gravitino'.                                                                                                                                                                                                                                                                                       | Yes      | 0.2.0         |
+| gravitino.metalake | string | (none)                | The `gravitino.metalake` defines the metalake used. You can create it beforehand or later on. If not set, Trino might throw an error upon startup. It must be set to a valid metalake name. If it's not created, the Gravitino connector continues checking until it's created. Once created, it will load the catalogs, schemas, and tables into Trino and maintain synchronization. | Yes      | 0.2.0         |
```
Comment (jerryshao) [inline docs/trino-connector/configuration.md:12]: "It must be set...", do not use passive voice.

File: docs/trino-connector/supported-catalog.md [hunk 4]
```diff
@@ -15,13 +15,12 @@ The catalogs currently supported by the Gravitino connector are as follows:
...
 
-Trino currently does not support creating Gravitino managed catalogs. 
-If you need to create a catalog, please refer to:
-- [Create a Catalog](../manage-metadata-using-gravitino.md#create-a-catalog)
+Trino currently does not support creating Gravitino managed catalogs, if you need to create a catalog, please refer to: [Create a Catalog](../manage-metadata-using-gravitino.md#create-a-catalog)
 
 ## Data type mapping between Trino and Gravitino
 
-All types of catalogs support the following data type conversions between Trino and Gravitino:
+Gravitino connector supports the following data type conversions between Trino and Gravitino currently. Depending on the detailed catalog, some data types conversion may not be supported for this specific catalog, for example,
```
Comment (jerryshao) [inline docs/trino-connector/supported-catalog.md:22]: Do not use the passive vocide "may not be supported"

**Comment by yuqi1129** [comment]
Everything has been resolved.

**Review by diqiu50 — APPROVED** [review]
LGTM

File: docs/trino-connector/configuration.md [hunk 5]
```diff
@@ -6,8 +6,8 @@ license: "Copyright 2023 Datastrato Pvt Ltd.
...
+| connector.name     | string | (none)                | The `connector.name` defines the name of Trino connector, this value is always 'gravitino'.                                                                                                                                                                                                                                                                                                                                         | Yes      | 0.2.0         |
+| gravitino.metalake | string | (none)                | The `gravitino.metalake` defines which metalake in Gravitino server the Trino connector uses. Trino connector should set it at start, the value of `gravitino.metalake` need to a valid name of metalake, no matter Gravitino has created it or not, once Trino connector successfully detect the existence of metalake, it will load all catalogs, schemas, and tables under this metalake automatically and keep synchronization. | Yes      | 0.2.0         |
```
Comment (jerryshao) [inline docs/trino-connector/configuration.md:12]: need to be a valid name..., Trino connector can detect and load the metalake with catalogs, schemas and tables once created and keep in sync.

File: docs/trino-connector/configuration.md [hunk 6]
```diff
@@ -6,8 +6,8 @@ license: "Copyright 2023 Datastrato Pvt Ltd.
...
+| gravitino.url      | string | http://localhost:8090 | The `gravitino.url` defines the connection URL of the Gravitino server, the default value is `http://localhost:8090`. If this Gravitino server has not started, Trino can still start and work normally. Once the Gravitino server is up and running, Trino connector will automatically connect to Gravitino sever and load metadata.                                                                                              | Yes      | 0.2.0         |
```
Comment (jerryshao) [inline docs/trino-connector/configuration.md:13]: Trino connector can detect and connect to Gravitino server once it is ready, no need start Gravitino beforehand.

File: docs/trino-connector/configuration.md [hunk 7]
```diff
@@ -6,8 +6,8 @@ license: "Copyright 2023 Datastrato Pvt Ltd.
...
+| connector.name     | string | (none)                | The `connector.name` defines the name of Trino connector, this value is always 'gravitino'.                                                                                                                                                                                                                           | Yes      | 0.2.0         |
+| gravitino.metalake | string | (none)                | The `gravitino.metalake` defines which metalake in Gravitino server the Trino connector uses. Trino connector should set it at start, the value of `gravitino.metalake` needs to be a valid name, Trino connector can detect and load the metalake with catalogs, schemas and tables once created and keep in sync.   | Yes      | 0.2.0         |
+| gravitino.url      | string | http://localhost:8090 | The `gravitino.url` defines the connection URL of the Gravitino server, the default value is `http://localhost:8090`. Trino connector can detect and connect to Gravitino server once it is ready, no need start Gravitino server beforehand.                                                                         | Yes      | 0.2.0         |
```
Comment (jerryshao) [inline docs/trino-connector/configuration.md:13]: Sorry, my bad. "no need to.."

Comment (yuqi1129) [reply docs/trino-connector/configuration.md:13]: OK

**Comment by jerryshao** [comment]
@yuqi1129 please rebase the code.

**Comment by yuqi1129** [comment]
> @yuqi1129 please rebase the code.

Done

**Pull request merged**